### ISSUE TITLE
Ask a question/catalog feedback navbar does not have sufficient color contrast

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/orangelight#3579]
### Steps to recreate

- Use WAVE, Lighthouse, or axe dev tools to scan a catalog show page
- Notice that the Ask a question/catalog feedback navbar does not have sufficient contrast between the blue link text and the grey background.

### URL

https://catalog.princeton.edu/catalog/SCSB-2312348 (and any other show page)

### Issues

[WCAG 1.4.3 Contrast Minimum](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html)

### Severity

Medium

### WCAG Conformance level (A, AA, AAA)

Level AA

### User impact

Makes it more difficult for patrons to notice and use these forms.

### Recommended fix

Change the background color, link text color, or both until they have a 4.5:1 contrast ratio.

### Acceptance criteria
- [ ] The link text color and background color in this menu have a 4.5:1 contrast ratio, as measured by a tool like WAVE, axe devtools, lighthouse, or [Color Contrast Analyzer](https://www.tpgi.com/cca-download/).
